includes/create-theme/theme-locale.php

```diff
@@ -28,6 +31,24 @@ private static function escape_text_content( $string ) {
 
 		$string = addcslashes( $string, "'" );
 
+		$p = new CBT_Token_Processor( $string );
+		$p->process_tokens();
+		$text             = $p->get_text();
+		$tokens           = $p->get_tokens();
+		$translators_note = $p->get_translators_note();
+
+		if ( ! empty( $tokens ) ) {
+			return "<?php $translators_note echo sprintf( esc_html__( '$text', '" . wp_get_theme()->get( 'TextDomain' ) . "' ), " . implode(
```

Comment:
I'm not certain, but I suspect the translators comment may need to be on a separate line, immediately above the `esc_html__` function, to be parsed correctly.